convert2rhel/utils.py

```diff
@@ -533,3 +533,13 @@ def backup(self):
 
 
 changed_pkgs_control = ChangedRPMPackagesController()  # pylint: disable=C0103
+
+
+def is_rpm_based_os():
```

Comment:
Since this function is used in tests only, it should be rather in `unit_tests/__init__.py`.

Reply:
solved in #151